app/pharmacy/controllers/dashboardController.js

```diff
@@ -5,13 +5,12 @@
     .module('pharmacy')
     .controller('DashboardController', DashboardController);
 
-  DashboardController.$inject = ['$state', '$stateParams', 'patientService'];
```

Comment:
It's not actually needed. I've removed it because I got tired off having to write dependencies twice every time. 

https://docs.angularjs.org/guide/di#implicit-annotation

There is no problem yet because our application code is not minified :/